### ISSUE TITLE
[Crash]: shoot at a bomb running on a brick

### ISSUE THREAD

**Issue by haehnchen72** [SuperTux/supertux#3122]
### SuperTux Version

SuperTux 2 (v0.6.3-1471-g725a20ad9

### System Information

Archlinux 

### Steps To Reproduce Crash

When a bomb runs over bricks with coins and is then shot down by tux, supertux crashes

### Debugging Information (Stacktrace)

```shell
supertux2(_ZN12ErrorHandler14get_stacktraceB5cxx11Ev+0x40) [0x598Tf5722b520] supertux2(_ZN12ErrorHandler12handle_errorEi+0x6b) [0x598757e2cbbb] /usr/lib/libe.so.6(+0x3d1d0) [0x7bf2a%eleidO] supertux2(_ZN5BlockiZstart_bounceEP12MovingObject+0x51) [0x598757e9ca51] supertux2(_ZN5Brick9try_breakEP6Playerb+0x2fa) [0x598f57d0c3ea] supertux2(_ZN9Explosion7explodeEv+0x7b5) [0x598757d57f25] supertux2(_ZN9Explosion6updateEf+Oxad) [0x598f57d5811d] supertux2(_ZN176ameObjectManager6updateEf+0x35) [0x598f57ae42e5] supertux2(_ZN6Sector6updateEf+0x1ie) [0x598f57b5e57e] supertux2(_ZN116ameSession6updateE fRKi1OController+0x1d5) [0x598f57af4915] supertux2(_ZN13ScreenManageribupdate_gamelogicEf+0x6a) [0x598f57b5a27a] supertux2(_ZN13ScreenManagerSloop_iterEv+0x1f5) [0x598f57b5b2d5] supertux2(_ZN13ScreenManager3runEv+0x30) [0x598f57b5b600] supertux2(_ZN4Mainiilaunch_gameERK20CommandLineArguments+Oxbéd) [0x5938f57b113dd] supertux2(_ZN4Main3runEiPPc+0x42b) [0x598f57b1294b]
supertux2(maintOx4e) [0x598f57a5773¢]
/usr/lib/libc.so.6(+0x25e08) [Ox7bf2a%e06e08] /usr/lib/libc.so.6(__libc_start_main+Ox8c) [Ox7bf2a%e06ecc] supertux2(_start+Ox2e) [0x598f57a8841e]
```

### Additional Information

https://github.com/user-attachments/assets/4be6ca67-8a8b-4ed4-8d40-0e536ae123a5

### Guidelines For Reporting Issues

- [X] I have read <https://github.com/SuperTux/supertux/blob/master/CONTRIBUTING.md#bug-reports>.
- [X] I have verified this isn't an issue that's already been reported.
- [X] I have verified this isn't a [discussion](https://github.com/SuperTux/supertux/discussions), or an issue about unintended behavior or a feature request, but rather an actual crash ─ that is, the game closed unexpectedly.
- [X] I have verified this issue is **not** about wrong translations (use Transifex for those), or anything unsupported (e.g. third-party add-ons).
- [X] In this report, I have only included details about **one** (1) crash.
- [X] If I make a mistake while submitting this report, I agree to use the "Edit" feature to correct it, instead of closing this issue and opening a new one.

**Comment by tobbi** [comment]
I couldn't reproduce the issue with latest master. Can you attach the level here?

Also, it would be nice if you could get the same stack trace using a debug build so the symbols aren't mangled.

**Comment by haehnchen72** [comment]
II use appimage

**Comment by Brockengespenst** [comment]
Shod be solved by #3118 .
